### ISSUE TITLE
Use any e-mail-address as the issuers address.

### ISSUE THREAD

**Issue by sveeke** [edubadges/audit#24]
When creating a new issuer only verified e-mail-addresses are listed. However it is possible to intercept the request and change it to a different non-verified e-mail-address.

**Comment by sveeke** [comment]
threatLevel="Low" type="Functionality Abuse"

When creating a new issuer only verified e-mail-addresses are listed. However it is possible to intercept the request and change it to a different non-verified e-mail-address.

**Default address:**

![image](https://user-images.githubusercontent.com/12334213/41537104-c269bdac-7307-11e8-8975-c520407ad26a.png)


**Edited request:**

```
PUT /v1/issuer/issuers/efOVSBKyQzqMrY5IPaHSiQ HTTP/1.1
Host: badgr-dev2.edubadges.nl
User-Agent: Mozilla/5.0 (Windows NT 6.3; rv:36.0) Gecko/20100101 Firefox/36.04
Accept: application/json
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Authorization: Token 5a29a471f3b21be11928361f5c42aeabf0c5cd8f
Content-Type: application/json
Referer: https://surf-dev2.edubadges.nl/issuer/issuers/efOVSBKyQzqMrY5IPaHSiQ/edit
Content-Length: 222
Origin: https://surf-dev2.edubadges.nl
Connection: close

{"name":"bla","description":"hjhjh","email":"oipipiplltralala@gmail.com","url":"https://www.edubadges.nl","image":"http://badgr-dev2.edubadges.nl/media/uploads/issuers/issuer_logo_d969e09d-a3b4-43b8-8fa4-ead933883123.png"}
```

**New address:**

![image](https://user-images.githubusercontent.com/12334213/41537125-ccfe8fa4-7307-11e8-9b4d-b494c078d332.png)

**Comment by ottonomy** [comment]
Hey, this looks like you're using a recent version of Badgr for this issue. :+1: A fix should be implemented.

**Comment by sveeke** [comment]
@ottonomy has this fix been implemented?

**Comment by ottonomy** [comment]
Yes, this has been fixed both in the UI and API now (previously the locking was just in the UI). There is a workaround if the server-operator needs a certain user to be able to create issuers via API without on-system email verification.
